docs/administrator/security/component-permission.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Component Permissions
+---
+
+# Karmada Component Permissions
+
+This document provides a detailed explanation of the resources each Karmada component needs to access and the reasons for these accesses. It aims to ensure secure management and compliant access to resources within the system. As a multi-cluster management framework, controlling permissions between Karmada components is essential for maintaining the security and stability of the entire system.
+
+Karmada uses [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to manage access to its components. Proper configuration of RBAC ensures that only authorized users and services can perform specific actions, enhancing security and operational integrity. This document will help administrators understand and configure the RBAC permissions needed for Karmada components effectively.
+
+## karmada-operator
+
+### Introduction:
+
+The Karmada operator is a method for installing, upgrading, and deleting Karmada instances. It builds upon the basic Karmada resource and controller concepts, provides convenience to centrally manage entire lifecycle of Karmada instances in a global cluster.
+
+### Access Permissions:
+
+#### Resources in Cluster Where karmada-operator is deployed
+
+##### Cluster-Scope:
```

Comment:
Are you going to classify the following resources by scope?
Note that the resources in the following table include both cluster-scoped and namespace-scoped.